omnibus_overrides.rb

```diff
@@ -2,6 +2,7 @@
 # When updating this, check doc/FrequentTasks.md for checklists to ensure all
 # the various usages are updated in lockstep
 #
+override :rabbitmq, version: "3.6.16"
```

Comment:
can we update to a more recent version of rabbitmq?

Reply:
Or create an issue to track using the latest version of rabbitmq.

Reply:
https://github.com/chef/chef-server/issues/1825